### ISSUE TITLE
Hand-only performance on AGORA.

### ISSUE THREAD

**Issue by mls1999725** [mks0601/Hand4Whole_RELEASE#19]
Hi, I tested the hand-only pre-trained model given by the repo on the AGORA validation set. The test error is far from the paper (The test results and the hand results of the paper are shown in the picture below). Could you please help me check if my hand-only test method is wrong? By the way, do you remember how long it took for the competition application on this website to pass?
![snapshot_12](https://user-images.githubusercontent.com/43207078/177007934-256bfc1b-d5b8-40fe-bd89-561d4021f251.PNG)
![image](https://user-images.githubusercontent.com/43207078/177007973-19f4122b-c4ad-4f43-8fee-ebef6e84c2a8.png)

Looking forward to your reply!

**Comment by mks0601** [comment]
The AGORA results in the table is from a whole-body network (hand4whole), not a hand-only network (pose2pose).
The codalab often gets stuck..

**Comment by mls1999725** [comment]
> The AGORA results in the table is from a whole-body network (hand4whole), not a hand-only network (pose2pose). The codalab often gets stuck..

Thanks ! Is this website unable to submit the results immediately after registration? After I registered, the website gave me this prompt "Your request to participate in this challenge has been received and a decision is pending".
